### PR TITLE
Configuration API for v1.23

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

File: genref/go.mod [hunk 1]
```diff
@@ -3,31 +3,33 @@ module github.com/kubernetes-sigs/reference-docs/genref
 go 1.16
```
Comment (kbhawkey) [inline genref/go.mod:3]: Should you build with Go 1.17?

File: genref/config.yaml [hunk 2]
```diff
@@ -44,18 +44,29 @@ apis:
     # Skip this because not referenced anywhere and it has no group name
     skip: true
 
+  - name: kubelet-credentialprovider
+    title: Kubelet CredentialProvider (v1alpha1)
```
Comment (kbhawkey) [inline genref/config.yaml:48]: At release time, how do you know what to add to config.yaml?

File: genref/go.mod (hunk 1 above)
Comment (tengqm) [reply genref/go.mod:3]: Tried, but failed. Intra-package and inter-package references are not correctly resolved in 1.17. Need some time to investigate.

File: genref/config.yaml (hunk 2 above)
Comment (tengqm) [reply genref/config.yaml:48]: I'm not aware of any solution that can automate the discovery of new versions.

**Comment by tengqm** [comment]
/hold
Reference target is not correct.

File: genref/config.yaml (hunk 2 above)
Comment (kbhawkey) [reply genref/config.yaml:49]: OK. I am trying to sort out how we can provide instructions for the release team (and others) to generate these reference pages.

**Comment by kbhawkey** [comment]
These changes look ready to merge. Thanks @tengqm .
/lgtm
Feel free to release the hold.
/hold

**Comment by tengqm** [comment]
Releasing the hold for follow up work.
/hold cancel

**Pull request merged**